Backend/app/routes/brand_dashboard.py

```
@@ -0,0 +1,1132 @@
+
+def validate_uuid_format(id_value: str, field_name: str = "ID"):
+    """Validate UUID format"""
+    if not id_value or len(id_value) != 36:
+        raise HTTPException(status_code=400, detail=f"Invalid {field_name} format")
+    return True
+
+def safe_supabase_query(query_func, error_message: str = "Database operation failed"):
+    """Safely execute Supabase queries with proper error handling"""
+    try:
+        result = query_func()
+        return result.data if result.data else []
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)